server.js

```diff
@@ -55,11 +57,8 @@ app.get('/*', (req, res) => {
       }
       const store = configureStore(res.data);
 
-      const application = ReactDOMServer.renderToString(
-        <Provider store={store}>
-          <RouterContext {...renderProps} />
-        </Provider>
-      );
+      const application = ReactDOMServer
+        .renderToString(<Provider store={store}><RouterContext {...renderProps} /></Provider>);
```

Comment:
I prefer to have these on multiple lines to look more like formatted HTML but not a big deal.

Reply:
the eslint rules made it like this when formating